osu.Desktop.VisualTests/Tests/TestCaseMultiUserPanel.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using OpenTK;
+using OpenTK.Graphics;
+using osu.Framework.Testing;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Screens.Select;
+using osu.Game.Screens.Multiplayer;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Primitives;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Input;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Users;
+
+namespace osu.Desktop.VisualTests.Tests
+{
+    class TestCaseMultiUserPanel : TestCase
+    {
+        private MultiUserPanel test;
+        private FillFlowContainer panelContainer;
+        public User User = new User
```

Comment:
This could be created in `Reset()`, rather than save it(see visual test of leaderboard)